framework/db/sqlite/QueryBuilder.php

```diff
@@ -394,6 +394,15 @@ public function build($query, $params = [])
             $sql = "$sql{$this->separator}$union";
         }
 
+        // replace aliases with table names
+        $sql = preg_replace_callback(
+            '/\\{\\{@([\w\-\. ]+)\\}\\}/',
+            function ($matches) use ($query) {
+                return '{{' . $query->getAlias($matches[1]) . '}}';
+            },
+            $sql
+        );
```

Comment:
Perhaps a separated method should be extracted.
Perhaps it should be somewhere at `yii\db\Schema` allong with `quoteTableName()` and `quoteColumnName()`.